CHANGELOG.md

```diff
@@ -8,11 +8,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
+- Support lua functions to for keymaps
+- Allow to disable existing keymaps by setting them to `nil`
```

Comment:
I wonder, have you actually tried doing this?

I believe that the way lua tables and `tbl_deep_extend` work means that doing `Presenting.setup { keymaps = { ['q'] = nil } }` is exactly equivalent to `… keymaps = {}`, and the <kbd>q</kbd> mapping thus remains in place. Luckily, `set_slide_keymaps` is defined in a way that any `type(v)` other than "string" or "function" is ignored, so `… keymaps = { ['q'] = 0 }` disables the <kbd>q</kbd> mapping, but this should probably be documented…

Reply:
Thanks, @liskin, for taking a look. I think I tried, but it's been a while. Maybe I did a last minute change.

If you think this needs fixing or you want to clarify this, PRs are always welcome!